daemon/src/taker_cfd.rs

```diff
@@ -60,6 +60,29 @@ enum RollOverState {
     None,
 }
 
+pub struct AutoRolloverParams {
+    settlement_interval: Duration,
+    /// Defines the minimum duration a CFD needs to be open for until a roll-over is triggered.
+    max_open_until_rollover: Duration,
```

Comment:
I find it confusing that the comment states `minimum time ...` but then the variable is called `max_...` 

This represents the `1 hour` a CFD has to be at least open until we trigger a rollover. I think the current name does not transport that meaning.

Reply:
My original suggestion called it `min_open_until_rollover` :D